database/postgreSQL/schema.sql

```diff
@@ -2,32 +2,72 @@ DROP DATABASE IF EXISTS playbar;
 
 CREATE DATABASE playbar;
 
-USE playbar;
+\c playbar;
 
-CREATE TABLE songs (
-  songId INT AUTO_INCREMENT,
-  likeCount INT DEFAULT 0,
+CREATE TABLE songs 
+(
+  songId serial PRIMARY KEY,
+  likeCount integer DEFAULT 0,
   songDataURL VARCHAR (150),
   songName VARCHAR (30),
   artist VARCHAR (30),
   album VARCHAR (30),
-  thumbnailURL VARCHAR (150),
-  PRIMARY KEY (songId)
+  thumbnailURL VARCHAR (150)
 );
 
-CREATE TABLE likes (
-  id INT AUTO_INCREMENT,
-  songId INT,
-  userName INT,
+CREATE TABLE likes 
+(
+  id serial PRIMARY KEY,
+  songId integer,
+  userName VARCHAR(30),
   FOREIGN KEY (songId)
     REFERENCES songs(songId)
-  PRIMARY KEY (id)
-)
+);
 
-CREATE TABLE playHistory (
-  id INT AUTO_INCREMENT,
-  songId INT,
-  userId INT,
+CREATE TABLE playHistory 
+(
+  id serial,
+  songId integer,
+  userName VARCHAR(30),
   FOREIGN KEY (songId)
     REFERENCES songs(songId)
+);
+
+INSERT INTO songs (
```

Comment:
assume you're going to remove the insertions / queries later

Reply:
Yes, those are for testing and will be removed for future use